compose/config/validation.py

```diff
@@ -64,6 +90,26 @@ def format_expose(instance):
     return True
 
 
+@FormatChecker.cls_checks("subnet_ip_address", raises=ValidationError)
+def format_subnet_ip_address(instance):
+    if isinstance(instance, six.string_types):
+        if '/' not in instance:
+            raise ValidationError("should be of the format 'IP_ADDRESS/CIDR'")
+
+        ip_address, cidr = instance.split('/')
```

Comment:
```
>>> format_subnet_ip_address('a/b/c')
ValueError: too many values to unpack
>>>
```

Since we never want more than 2 parts, we should do `instance.split('/', 1)` here.